src/lib/features/project/project-service.e2e.test.ts

```diff
@@ -962,7 +962,7 @@ test('should not change project if feature flag project does not match current p
     }
 });
 
-test('should return 404 if no project is found with the project id', async () => {
+test('should return 404 if no active project is found with the project id', async () => {
```

Comment:
looks like 2 tests

Reply:
I wanted to avoid setting up the same thing twice